src/main/java/com/getindata/connectors/http/internal/sink/httpclient/JavaNetSinkHttpClient.java

```diff
@@ -106,7 +106,7 @@ private SinkHttpClientResponse prepareSinkHttpClientResponse(
                         optResponse.orElse(null), sinkRequestEntry, endpointUrl, headerMap);
             } catch (PostRequestCallbackException e) {
                 failedCallback = true;
-                log.info("request marked as failed due to callback exception", e);
+                log.debug("request marked as failed due to callback exception", e);
```

Comment:
should this be error?

Reply:
My goal behind raising a PostRequestCallbackException is to treat this particular request as failed. Any user who intentionally throws this exception would already be aware of the error from that particular request, so it doesn't feel like there's a need to log higher than debug.

Based on your comments on the PR, I think the naming of that exception class need to be iterated on, will give it a think 👍

Reply:
Renamed to FailedRequestException, lmk if this makes more sense 🙏